projects/1.20/assets/ex-deorum/exdeorum/lang/zh_cn.json

```
@@ -0,0 +1,171 @@
+{
+    "advancements.exdeorum.core.barrel.description": "使用木桶将有机物堆肥以形成泥土",
+    "advancements.exdeorum.core.barrel.title": "那是绿色垃圾",
+    "advancements.exdeorum.core.crook.description": "制作一个钩子，用它从树叶上收获更多的树苗。",
+    "advancements.exdeorum.core.crook.title": "给他钩子",
+    "advancements.exdeorum.core.root.description": "在虚空世界中诞生，眼前只有一座小岛",
+    "advancements.exdeorum.core.root.title": "别往下看...",
+    "advancements.exdeorum.core.silk_worm.description": "获得一个蚕，然后用它感染一棵树，得到线",
+    "advancements.exdeorum.core.silk_worm.title": "这看起来可以吃",
+    "advancements.exdeorum.core.string_mesh.description": "制作一个线网，将它放在筛子上以使用。",
```

```suggestion
    "advancements.exdeorum.core.string_mesh.description": "制作一个用于筛子上的线网。",
```
`string mesh`这个是怎么用于`sieve`的？

---

呃，不是说了放在筛子上。。还是说要写用右键放在？

---

我是说，是用于合成/铁砧，还是右键使用

---

ok，我写清楚点。